db/migrate/20171024074942_convert_autograded_text_response_answers_to_plaintext.rb

```diff
@@ -0,0 +1,18 @@
+class ConvertAutogradedTextResponseAnswersToPlaintext < ActiveRecord::Migration[5.0]
+  # For autograded text response questions, a plaintext textarea input box is shown instead of
+  # the rich text editor.
+  #
+  # Convert existing answers to autograded questions from HTML to plaintext.
+  def change
```

Comment:
Does having `change` here allow for us to rollback? I reckon what might happen is only that the schema version is reverted, but the migration has no way of know how to transform back.

Reply:
No, we can't rolllback as the formatting is lost. I've changed it to throw an error instead, as suggested by the docs.